pdg/src/info.rs

```diff
@@ -0,0 +1,204 @@
+use std::collections::HashMap;
+use crate::graph::{Graphs,GraphId,Graph,NodeId,Node,NodeKind};
+use rustc_index::vec::IndexVec;
+use rustc_middle::mir::Field;
+use std::collections::HashSet;
+use std::fmt::{self,Debug, Formatter, Display};
+
+#[derive(Debug, Eq, PartialEq, Hash, Clone)]
+pub struct NodeInfo {
```

Comment:
Is `NodeInfo` a descriptive enough name for this?  Did you have any other ideas in mind?  What do others think?